### PR TITLE
Fix cache keys in operating modes

### PR DESCRIPTION
**Pull request by ChrisAntaki**
b/236934384

### CODE REVIEW

File: src/runtime/services.ts [hunk 1]
```diff
@@ -80,7 +83,7 @@ const QUAL: OperatingMode = {
   frontEnd: 'https://subscribe-qual.sandbox.google.com',
   payEnv: 'SANDBOX',
   playEnv: 'STAGING',
-  feCache: CACHE_KEYS.hr1,
+  feCache: 'hr1',
```
Comment (jpettitt) [inline src/runtime/services.ts:86]: Why did you leave it caching ion Qual?

**Comment by fikus** [comment]
Can you add detail or reference a bug describing this change? I'm not sure what the intention of the change is.

**Comment by jpettitt** [comment]
> Can you add detail or reference a bug describing this change? I'm not sure what the intention of the change is.

b/236934384

File: src/runtime/services.ts (hunk 1 above)
Comment (ChrisAntaki) [reply src/runtime/services.ts]: Updated!

**Review by jpettitt — APPROVED** [review]

**Pull request merged**